### ISSUE TITLE
[Enhancement] Add a notice to README.md and if possible change automate.sh respectively

### ISSUE THREAD

**Issue by GithubUser699** [angela-d/nautilus-right-click-new-file#3]
Your README and script assumes that the language is english instead of every other language possible.
In order to make it work for everyone it would be good to point out this difference.

For example your script looks for the ~/Templates folder but in other languages this folder won't work because it is being translated.
In _NL_ it might be `~/Sjabloon` or in _GER_ it might be `~/Vorlagen`.
So your files (those which should be inside the right click menu) must be placed in this translated folder.

I don't know, unfortunately, where those translations are beeing stored but I guess there might be a file like those in /usr/share/desktop-directories.

**Comment by angela-d** [comment]
If you would like to extend this, I welcome a pull request!

Manual (english-focused) instructions are [here](https://github.com/angela-d/nautilus-right-click-new-file#optional-manual-instructions)
> Move the extracted files (templates) into /home/youruser/Templates/

Do you have `/home/youruser/Sjabloon`?

Or try:
```bash
cd ~/Sjabloon && pwd
```
^ That should give your Dutch version of `~/Templates`

**Comment by GithubUser699** [comment]
I'm sorry, I should have been a bit more clear here.

I know that your manuals and your script are english-focused and I moved the files accordingly.
My system runs mainly in german so my path would be `/home/user/Vorlagen/`

However, what I meant is that some people might want to create `~/Templates/` and copy the files into it and are confused that it isn't working as expected, because they had to copy it to their translated path, e.g. `~/Vorlagen` instead of the english _Templates_ path.

So basically I just wanted to point out that a note on your manuals for non-english users would be great and that it would be even greater if the script would automatically move it to the correct folder (which means it should be able to determine the users language but I don't know how to accomplish this).

Pull request should be coming soon if I do it right.

**Comment by GithubUser699** [comment]
I solved it. You should test it yourself, but for me it worked and it sounds logical to me, that it works for every other supported language as well.
See pull request.

**Comment by angela-d** [comment]
Please add the conditional check and I'll run a test on my system, after!

**Comment by GithubUser699** [comment]
A little extra to my comments in the pull request section. To get the value - which I assigned to customPath - for your system run

`source ~/.config/user-dirs.dirs && echo $XDG_TEMPLATES_DIR`

It should give you something like `/home/MyUser/Templates`

(I'd give you a screenshot but github says "Something went really wrong, and we can't process that file. Try again." even though it's just a png screenshot file. Strange...)

EDIT: It works now. Github needs amazonaws.com to upload pictures...
Here it is:
![Screenshot_1](https://user-images.githubusercontent.com/74927253/105170310-acb5c800-5b14-11eb-9f09-a00d8be00d91.png)
Ignore the ^C.

**Comment by angela-d** [comment]
Works as expected!
Thanks for your efforts. :)
